### ISSUE TITLE
Product list: Row actions div is rendered in thumb column

### ISSUE THREAD

**Issue by gedex** [woothemes/woocommerce#8578]
### To reproduce:

* Make sure to use WP 4.3+. This is happened per https://github.com/WordPress/WordPress/commit/4c0c7fd7 -- where concept of a "primary" column is introduced. The fallback of primary is using the first non-checkbox column.
* Go to `/wp-admin/edit.php?post_type=product`
* You should get following row actions when product row is hovered:

  <img width="597" alt="screen shot 2015-07-15 at 7 16 43 am" src="https://cloud.githubusercontent.com/assets/78313/8688114/f484654e-2ac1-11e5-8a9d-b0231b104f2b.png">

**Comment by claudiosmweb** [comment]
Confirmed testing WordPress master. 

But let's hold this a little, to wait for a beta or RC, so sure if we need to fix something or not.

**Comment by jsd-aaron** [comment]
I'm using WordPress 4.3, and I'm getting this exact same error. Duplicate row actions, exactly like your screenshot. Is there a fix coming for this?

**Comment by gedex** [comment]
@jsd-aaron Which version of WooCommerce installed on your site? The issue has been fixed in WooCommerce 2.4.

**Comment by oreosmakemehappy** [comment]
I'm having the same issue.  Since WooCommerce 2.4 seriously hosed my attributes AND my shipping structure when I tried updating, and I had to go back to 2.3.13 and won't be able to update until I have time to manually go through hundreds of products and fix them one by one, I'd rather just get a fix for this issue.  But it looks like there is no quick fix other than turning the image column off in the admin panel, which I will do until then.
